Gruntfile.js

```diff
@@ -26,6 +26,23 @@ grunt.initConfig({
 			options: { process: process },
 			src: [
 				"src/intro.js",
+				"src/core/utilities.js",
+				"src/core/extend.js",
+				"src/core/objectType.js",
+				"src/core/urlParams.js",
+				"src/core/stacktrace.js",
+				"src/core/config.js",
+				"src/core/defined.js",
+				"src/core/loggingCallbacks/register.js",
+				"src/core/loggingCallbacks/run.js",
+				"src/core/loggingCallbacks/verify.js",
+				"src/core/control/begin.js",
+				"src/core/control/process.js",
+				"src/core/control/pauseProcessing.js",
+				"src/core/control/resumeProcessing.js",
+				"src/core/control/done.js",
+				"src/core/module.js",
+				"src/core/test.js",
```

Comment:
This went deep, and I don't believe it's better to make this split into many files.

There're some things I liked, like the errorString function going to the assert.js file.

Maybe a better approach, using smaller files:

- `src/core/module.js`: it doesn't exist.
- `src/core/test.js`: may go to `src/test.js`
- `src/core/control/*` they can stay at `core.js` 
- `src/core/loggingCallbacks/*` => `src/core/logging.js` would be fine
- can create a `src/utilities.js` containing:
  - `src/core/utilities.js`
  - `src/core/extend.js`
  - `src/core/objectType.js`
  - `src/core/urlParams.js` 
  - `src/core/defined.js`
- `src/core/config.js` is fine